### DISCUSSION TITLE
Release 7.24.1

### DISCUSSION THREAD

**Discussion by matteo-mancini** [babel/babel#16364] [Releases] [answered]
Hi to all, i just tried to install packege by npm but i am getting this error:

`npm i -g @babel/highlight@^7.24.1
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @babel/highlight@^7.24.1.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.`

Could you please verify public availability of artifact ?
Thanks

**Comment by liuxingbaoyu** [comment, marked as answer]
Hi!
This seems to be an issue caused by a package that you are not able to download. You can verify that the package is available using the npm website: for example, https://www.npmjs.com/package/@babel/types?activeTab=versions is the URL for `@babel/types` (replace `@babel/types` in the URL with the package you want to check). If you see zero downloads don't worry: npm can take up to a few days to display the downloads count, but this does not mean that the package is not available.
Here are some things you can try:
1. Please check if there is a stale proxy or cache between you and npm (this is the most likely cause and it is common in corporate networks).
2. Visit https://status.npmjs.org/ to see if the current npm is working properly.
3. See https://gist.github.com/hzoo/df94af2d2ec48696e683b772c775aa67

https://www.npmjs.com/package/@babel/highlight?activeTab=versions

**Comment by matteo-mancini** [comment]
Hi and thanks for your reply,
Of course, all you suggest is very useful but are all things I already checked.

***@***.***


As for the proxy we use Nexus, but even bypassing it , the package does not come in..


Matteo

**Comment by matteo-mancini** [comment]
Solved !
Actually, i did not fully bypass my nexus, once without nexus it worked !
Thanks again!

Matteo Mancini

From: Matteo Mancini
Sent: martedì 19 marzo 2024 16:28
To: babel/babel ***@***.***>; babel/babel ***@***.***>
Cc: Author ***@***.***>
Subject: RE: [babel/babel] Release 7.24.1 (Discussion #16364)

Hi and thanks for your reply,
Of course, all you suggest is very useful but are all things I already checked.

***@***.***


As for the proxy we use Nexus, but even bypassing it , the package does not come in..


Matteo
